webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -2867,6 +2871,10 @@ def mocked_get(url, params, **options):
                     'signature': 'FakeSignature1',
                     'uuid': '11cb72f5-eb28-41e1-a8e4-849982120611',
                     'process_type': None,
+                    'os_name': 'Windows NT',
+                    'product': 'WaterWolf',
+                    'version': '1.0',
+                    'cpu_name': 'amd64',
```

Comment:
Because this test renders the report index page and the new bugzilla_submit_url there reads from the `report` dict which now needs those keys to exist.